ciphers/ColumnarTranspositionCipher.java

```diff
@@ -1,168 +1,198 @@
+            return word.length() / keyword.length();
         }
     }
-    private static Object[] findElements(){
-        Object[] charValues=new Object[keyword.length()];
-        for(int i=0;i<charValues.length;i++){
-            for(int j=0;j<abecedarium.length();j++){
-                if(keyword.charAt(i)==abecedarium.charAt(j))charValues[i]=j;
+
+    private static Object[] findElements() {
+        Object[] charValues = new Object[keyword.length()];
+        for (int i = 0; i < charValues.length; i++) {
+            for (int j = 0; j < abecedarium.length(); j++) {
+                if (keyword.charAt(i) == abecedarium.charAt(j)) {
+                    charValues[i] = j;
+                }
             }
         }
         return charValues;
     }
-    private static Object[][] sortTable(Object[][] table){
-        Object[][] tableSorted=new Object[table.length][table[0].length];
-        for(int i=0;i<tableSorted.length;i++){
-            for(int j=0;j<tableSorted[i].length;j++){
-                tableSorted[i][j]=table[i][j];
-            }
+
+    private static Object[][] sortTable(Object[][] table) {
+        Object[][] tableSorted = new Object[table.length][table[0].length];
+        for (int i = 0; i < tableSorted.length; i++) {
+            System.arraycopy(table[i], 0, tableSorted[i], 0, tableSorted[i].length);
         }
-        for(int i=0;i<tableSorted[0].length;i++){
-            for(int j=i+1;j<tableSorted[0].length;j++){
-                if((int)tableSorted[0][i]>(int)table[0][j]){
-                    Object[] column=getColumn(tableSorted,tableSorted.length,i);
-                    switchColumns(tableSorted,j,i,column);
+        for (int i = 0; i < tableSorted[0].length; i++) {
+            for (int j = i + 1; j < tableSorted[0].length; j++) {
+                if ((int) tableSorted[0][i] > (int) table[0][j]) {
+                    Object[] column = getColumn(tableSorted, tableSorted.length, i);
+                    switchColumns(tableSorted, j, i, column);
                 }
             }
         }
         return tableSorted;
     }
-    private static Object[] getColumn(Object[][] table,int rows,int column){
-        Object[] columnArray=new Object[rows];
-        for(int i=0;i<rows;i++){
-            columnArray[i]=table[i][column];
+
+    private static Object[] getColumn(Object[][] table, int rows, int column) {
+        Object[] columnArray = new Object[rows];
+        for (int i = 0; i < rows; i++) {
+            columnArray[i] = table[i][column];
         }
         return columnArray;
     }
-    private static void switchColumns(Object[][] table, int firstColumnIndex,int secondColumnIndex,Object[] columnToSwitch){
-        for(int i=0;i<table.length;i++){
-            table[i][secondColumnIndex]=table[i][firstColumnIndex];
-            table[i][firstColumnIndex]=columnToSwitch[i];
+
+    private static void switchColumns(Object[][] table, int firstColumnIndex, int secondColumnIndex, Object[] columnToSwitch) {
```

Comment:
Please break this line up. max. 80 columns

Reply:
:+1:  done